joblib/test/test_parallel.py

```diff
@@ -10,6 +10,10 @@
 import sys
 import io
 import os
+import subprocess
+from threading import Timer
+import re
```

Comment:
Second point in the imports part, under `Imports should be grouped in the following order: ` But after reading carefully, you are right saying there's no specific rule for having `from module import function` after `import module`. This was a mis-interpretation on my side.